app/__init__.py

```diff
@@ -19,6 +20,10 @@ def create_app(config):
 
     db.init_app(app)
     migrate.init_app(app, db)
+    
+    # setup flask_jwt_extended for authentication
+    app.config['JWT_SECRET_KEY'] = app.config['SECRET_KEY']
```

Comment:
should we error with some helpful message if `SECRET_KEY` doesn't exist in the config, or at least print out an angry message if the value is the default `12345`?

Reply:
Good idea. I added an assertion that will make the server fail to launch if the default `SECRET_KEY` is used. It doesn't fire in the development or testing environments so it shouldn't annoy us when we're working locally.